pkgs/games/super-tux-kart/default.nix

```diff
@@ -1,27 +1,26 @@
 { stdenv, fetchFromGitHub, fetchsvn, fetchpatch, cmake, pkgconfig
 , openal, freealut, libGLU_combined, libvorbis, libogg, gettext, curl, freetype
-, fribidi, libtool, bluez, libjpeg, libpng, zlib, libX11, libXrandr }:
+, fribidi, libtool, bluez, libjpeg, libpng, zlib, libX11, libXrandr, enet }:
 
 let
   dir = "stk-code";
 
 in stdenv.mkDerivation rec {
-  name = "supertuxkart-${version}";
-
-  version = "0.9.3";
+  pname = "supertuxkart";
```

Comment:
What happened to ```name```?

Reply:
Name is generated from pname/version

Reply:
Do you have a link to a reference for this? (slightly off-topic for this PR obvoiusly)

Reply:
python used to do it and it was extended to stdenv.mkDerivation. You can check in a repl (don't have the reference). There are a few packages in nixpkgs already taking advantages of this. Sry I can't find the PR right now.

Reply:
:+1: